### ISSUE TITLE
Minimum PowerShell version 5.1

### ISSUE THREAD

**Issue by thebigmoosey** [Azure/azure-powershell#5745]
### Description

Your [powershellgallery](https://www.powershellgallery.com/packages/AzureRM.Resources/5.4.0) states that the "Minimum PowerShell version" required is 3.0, but we've found we required a minimum version of 5.1.

### Script/Steps for Reproduction

With PowerShell 3.0 installed, we were getting failures calling `Login-AzureRmAccount`:

```
Login-AzureRmAccount : The term 'Login-AzureRmAccount' is not recognized as the name of a cmdlet, function, script file, or operable program. Check the spelling of the name, or if a path was included, verify that the path is correct and try again. 
```

Once we installed PS 5.1, these errors stopped occurring.

Can you please confirm whether this should be the case, and if so could you update the minimum required version to avoid future confusion?

**Comment by benPearce1** [comment]
I was also able to recreate this issue using a 2012 R2 Server with latest updates, and installing the Azure Powershell 5.4.0 MSI.

Got the following output:
`C:/>Login-AzureRmAccount
WARNING: PowerShell version 3 and 4 will no longer be supported starting in May 2018. Please update to the latest
version of PowerShell 5.1
Login-AzureRmAccount : The term 'Login-AzureRmAccount' is not recognized as the name of a cmdlet, function, script
file, or operable program. Check the spelling of the name, or if a path was included, verify that the path is correct
and try again.
At line:1 char:1
+ Login-AzureRmAccount
+ ~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : ObjectNotFound: (Login-AzureRmAccount:String) [], CommandNotFoundException
    + FullyQualifiedErrorId : CommandNotFoundException` 

$env:PSModulePath:
`
C:\Users\user\Documents\WindowsPowerShell\Modules;C:\Program Files\WindowsPowerShell\Modules;C:\Windows\system32\WindowsPowerShell\v1.0\Modules\;C:\Program Files (x86)\Microsoft SQL Server\120\Tools\PowerShell\Modules\;C:\Program Fil
es\WindowsPowerShell\Modules\;C:\Program Files (x86)\Microsoft SDKs\Azure\PowerShell\ResourceManager\AzureResourceManager\;C:\Program Files (x86)\Microsoft SDKs\Azure\PowerShell\ServiceManagement\;C:\Program Files (x86)\Microsoft SDKs\Azure\PowerShell\Storage\
`

**Comment by hrboyceiii** [comment]
+1
My customer ran into this 14 Mar 2018 as well.  We downloaded the 5.3.0 MSI and found aliases were broken; tried to download 5.5.0, would not install on Server 2012 R2 and PowerShell 4.0.

Is there a link to timeline for deprecating < PowerShell 5.x?  My customer has significant investment in automation and that deprecation has a huge impact on regression testing for them.

**Comment by maddieclayton** [comment]
The issue with alias importing was a bug in the 5.3.0 and 5.4.0 versions, and was fixed in 5.5.0.  We have deprecated support for PowerShell 3 and 4 in the MSI starting in version 5.5.0, and will deprecate support for PowerShell 3 and 4 altogether in May.

There a few possible workarounds for this issue: 
1) Install 5.5.0 from PSGallery (https://www.powershellgallery.com/packages/AzureRM/5.5.0).  This version will have the fix for aliases.
2) Change the usage of the alias "Login-AzureRmAccount" to the actual command "Connect-AzureRmAccount"
